### PR TITLE
Set up a 'Portfolio: setup and mobile first Project'

### PR DESCRIPTION
**Pull request by kifle23**
A new branch is created and made the required adjustments based on the tasks given here ->
https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md

### CODE REVIEW

File: README.md
Comment (AHP15) [inline README.md:1]: - [ ] Kindly change the MIT link to be the URL of the `LICENSE` file in your repo because it contains your repo's license

**Review by AHP15 — CHANGES_REQUESTED** [review]
Hi @kifle23 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
soclose

## Highlights
✅ Good GitHub Flow
✅ No Linter errors
✅ Great design

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md
Comment (AbelG101) [inline README.md:1]: _The comments here are all OPTIONAL_
- [OPTIONAL] Hey great job setting up your project but I'd like to apologize on behalf of the previous reviewer as he forgot to mention some of the links (Run tests & and Deployment) on the readme, table of contents section doesn't work as you've removed them from the body, please make sure that you you have all the main headlines in you're readme the only optional headline here is the FAQ and Live demo (if the project doesn't require a live demo link).
    - So please make sure to add them and If you don't have anything to include in that section you can for example write for the Run tests section: No tests here... 💯 and so on for the rest as well.
- [OPTIONAL] I've also noticed your tech stack section doesn't have the appropriate tech stacks you're using for the project so please make sure to update that as well, it misleads the viewer into thinking you've used Node.js when you in fact haven't.
- [OPTIONAL] Also please make sure to add a detailed summary of what you've achieved in this particular milestone in your PR description. 

    > A new branch is created and made the required adjustments based on the tasks given here -> https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md
    - The above description isn't detailed enough to give us an understanding of what you've done for this milestone 👍

**Review by AbelG101 — APPROVED** [review]
## Hi @kifle23 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it. 
Congratulations! 🎉 

## Highlights
- Correct use of GitHub Flow ✔
-  Well-structured files ✔
- All linter checks are passing ✔
- Descriptive PR title with a great summary ✔

## Optional suggestions
 - Hey Kifle, great job making the changes requested. I've also left some comments below that were missed by the previous reviewer try to fix them 👍.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**